docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
+<!-- RUN-10639/RUN-11389 - Researcher Service Refactoring -->
+* Improved configuration for authentication in the cluster. Configuring the Kubernetes cluster API server for authentication is no longer needed. This applies to environments where end users are using the UI only and not the CLI.
 
-<!-- RUN-10802 Quota management dashboard post-release -->
+<!-- RUN-12505/RUN-12506 - Support Kubeflow notebooks for scheduling/orchestration -->
+* Improved support for KubeFlow Notebooks. Now Run:ai supports scheduling of Kubeflow notebook CRDs with fractional GPUs. Kubeflow notebooks are identified automatically and use a special icon in the *Jobs* UI.
 
-<!-- RUN-10862/RUN-10863 Department as a workspace asset creation scope - phase 2-->
+<!-- RUN-10251/RUN-10252 - Block over-subscription of quota by Projects/Departments- -->
+* Improved quota settings by providing the ability to block over-subscription of quota by *Projects* or *Departments*. For more information, see [Over quota blocking](../Researcher/scheduling/the-runai-scheduler.md#limit-quota-over-or-under-subscription).
 
-<!-- RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page -->
+<!-- RUN-10404/RUN-11747 Submit distributed training
+* RUN-11194/RUN-11239 All changes done in the UI for distributed training are hidden behind feature flag 
+* RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page 
+* RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page
+* RUN-11219/RUN-11602 Submitting an MPI distributed training from UI - 3rd form page
+* RUN-11231/RUN-11240 Environment for distributed training
+-->
+* Added support for distributed training in the following:
 
-<!-- RUN-11194/RUN-11239 All changes done in the UI for distributed training are hidden behind feature flag -->
+      * Trainings form
+      * Environments form
+  
+    You can select `single` or `multi-node (distributed)` training. When configuring distributed training, you will need to select a framework from the list. For *Trainings* configuration, see [Adding trainings](../Researcher/user-interface/trainings.md#adding-trainings). See your Run:ai representative to enable this feature. For *Environments* configuration, see [Creating an Environment](../Researcher/user-interface/workspaces/create/create-env.md#creating-a-new-environment).
 
-<!-- RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page-->
+<!-- ADDLINK RUN-10411/RUN-11390 Support self-signed certificates-->
+* Run:ai can be installed in an isolated network. In this air-gapped configuration, the organization will not be using an established root certificate authority{target=_blank}. Instead, the organization creates a local certificate which serves as the root certificate for the organization. The certificate is installed in all browsers within the organization. For more information, see [link here](##org-cert.md).
 
```

Comment:
@jasonnovichRunAI we need to add another feature here https://runai.atlassian.net/browse/RUN-12943 
```suggestion
<!-- RUN-12943 Ability to configure cluster routes certificate in OpenShift-->
On OpenShift, you can now configure the certificate to be used in the cluster routes created by Run:ai, instead of using the OpenShift's certificate. See ##customize-cluster-install.md
```

Reply:
@jasonnovichRunAI looks good. However, the link to customize-cluster-install.md is not "Cluster route certificates", it should be called "Customize Cluster Installation"

Reply:
@gal-revach If there is a specific section for it in the master article, I reference that. If you reference the whole article (that is from the title) the reader will get confused or lost. Also, this file is the last file to be merged in the docs review, so this is more of a placeholder for me. This is why I have the "ADDLINK" in the comment (TODO list plugin for VSCode).

Reply:
@jasonnovichRunAI there is no dedicated section for it in the article, there is a new field added to the table.. so I'm not sure we can reference specifically to it - if we can, let's do so. In any case, I think it would be wrong to call the link "Cluster route certificates"